lib/persisters/s3.js

```diff
@@ -25,7 +25,17 @@ class S3 {
    */
 
   load() {
-    return this.lambda.get(this.bucket, this.key)
+    return new Promise((resolve, reject) => {
+      this.lambda.get(this.bucket, this.key)
+        .then(resolve)
+        .catch((e) => {
+          if (String(e).indexOf('NoSuchKey') > -1) {
```

Comment:
I don't think Lester unsubscribed, I think it's a collaborators/teams issue. I'll add this repo to `services`. sounds good to stick with default empty string for now